.gitignore

```diff
@@ -17,3 +17,5 @@ db.sqlite3
 # Media files will be served from Amazon S3 on Heroku
 /baggysoapshop/media/
 /baggysoap.log
+
+holdingPen.txt
```

Comment:
What is this file? It's good to add things to .gitignore instead of committing them, but only if they belong in the workspace... perhaps this does? I just have no idea what it is! :slightly_smiling_face:

Reply:
If it's something that you want to have in the workspace, but that others are unlikely to have, you can exclude it locally by adding it to `.git/info/exclude` :+1:

Reply:
Holdingpen.txt  is just a place I store bits of code I'm working on, it's odd as I did put it in the .gitignore very strange.